app/src/ui/branches/branch-list.tsx

```diff
@@ -231,15 +231,11 @@ export class BranchList extends React.Component<
   }
 
   private onRenderNewButton = () => {
-    if (this.props.canCreateNewBranch) {
-      return (
-        <Button className="new-branch-button" onClick={this.onCreateNewBranch}>
-          {__DARWIN__ ? 'New Branch' : 'New branch'}
-        </Button>
-      )
-    } else {
-      return null
-    }
+    return this.props.canCreateNewBranch ? (
```

Comment:
Using ternary because of the example in our [engineering values](https://github.com/desktop/desktop/blob/master/docs/contributing/engineering-values.md#immutability-is-good). However, I'm not against switching to normal `if/else` statements for the sake of readability. I do like the succinctness of this though.